### PR TITLE
Provides sample systemd env file, and organizes init system files

### PR DESCRIPTION
**Pull request by cluelessperson**

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (bauerj) [inline .idea/vcs.xml]: As much as I like IntelliJ, this should probably not be in the repository.

File: samples/init system/systemd/electrumx.conf [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    USERNAME = electrumx
+    ELECTRUMX = /usr/local/bin/electrumx_server.py
+
+    #Bitcoin Node Credentials
+    DAEMON_URL = http://username:password@hostname:port/
+        #Alternatively you may specify the individual variables
+        #DAEMON_USERNAME =
+        #DAEMON_PASSWORD =
+        #DAEMON_HOST =
+        #DAEMON_PORT = #optional, will default to coin appropriate default
```
Comment (bauerj) [inline samples/init system/systemd/electrumx.conf]: This is already obsolete :no_mouth:

File: samples/init system/systemd/electrumx.conf [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    #REPORT_TCP_PORT = $TCP_PORT
+    #REPORT_SSL_PORT = $SSL_PORT
+
+
+#
+#OPTIONAL VISIBILITY
+#
+    #BANNER_FILE = /path/to/banner
+    #DONATION_ADDRESS =
+    #IRC = freenode
```
Comment (bauerj) [inline samples/init system/systemd/electrumx.conf]: IRC=yes would also work 😉

Comment (bauerj) [inline samples/init system/systemd/electrumx.conf]: To use this, you will also need to install to `irc` package.

**Comment by kyuupichan** [comment]
Just 2 things:

- can you drop the .gitignore change; it seems to be something not relevant to most users
- can you drop the "init system" directory?  I don't want spaces in paths, but I don't think we need that level at all.  Just samples/daemontools and samples/systemd would be good

**Comment by kyuupichan** [comment]
Dup of #36 which went in

**Pull request closed without merging**